src/main/java/com/plugish/woominecraft/Commands/WooCommand.java

```diff
@@ -51,4 +47,12 @@ public boolean onCommand( CommandSender sender, Command command, String label, S
 		}
 		return true;
 	}
+
+	private String getChatPrefix() {
+		return chatPrefix;
+	}
+
+	private WooMinecraft getPlugin() {

```

Comment:
Also, this function is private, so I'm unclear on scope here, looks like this function is also used in `BukkitRunner.java` from my understanding of scope Private means it can only be ran in the current class, and therefore is inaccessible from other classes. ( I could be wrong here, this is java, not PHP ).

Even if we were to keep this function, shouldn't it be public and reside solely in the main plugin file itself, not in the commands file?